heartbeat/heartbeat.py

```diff
@@ -23,5 +23,10 @@
         print("Variable TEMPLATE_ID is missing")
     for base_url in BASE_URL:
         notifications_client = NotificationsAPIClient(API_KEY, base_url=base_url)
-        response = notifications_client.send_email_notification(email_address=EMAIL_ADDRESS, template_id=TEMPLATE_ID)
-        print("Email has been sent!")
+        try:
+            response = notifications_client.send_email_notification(email_address=EMAIL_ADDRESS, template_id=TEMPLATE_ID)
+            print("Email has been sent by {}!".format(base_url))
+        except Exception as e:
+            raise e
```

Comment:
Hmm it would not be necessary to catch the exception if we simply raise it next, right?

Reply:
true, I just prefer this stylistically, but I can change it to just `raise` if you prefer

Reply:
We could catch a `HTTPError`, print some error message and exit the process with a non `0` exit code. [Per documentation, we could have the response code when an error occurs](https://docs.notifications.service.gov.uk/python.html#error-codes).

If the exception is not a `HTTPError`, we can let it bubble up the stack and crash the program probably.

Reply:
> I can change it to just raise if you prefer

You don't even need to raise, the exception will occur and bubble up.